src/asm.ts

```diff
@@ -137,7 +137,7 @@ export class AsmParser {
     cudaBeginDef = /\.(entry|func)\s+(?:\([^)]*\)\s*)?([$.A-Z_a-z][\w$.]*)\($/;
     cudaEndDef = /^\s*\)\s*$/;
 
-    asmOpcodeRe = /^\s*(?<address>[\da-f]+):\s*(?<opcodes>([\da-f]{2} ?)+)\s*(?<disasm>.*)/;
+    asmOpcodeRe = /^\s*(?<address>[\da-f]+):([|\\\/>X-]*\s*)*(?<opcodes>([\da-f]{2} ?)+)\s*(?<disasm>.*)/;
```

Comment:
Looks like `+` sign is also a possible option. I've built a small binary which produces following objdump:
```
main():
/tmp/test.c:5
  401136:                 55                    push   %rbp
  401137:                 48 89 e5              mov    %rsp,%rbp
  40113a:                 48 83 ec 10           sub    $0x10,%rsp
/tmp/test.c:8
  40113e:       /-------> 48 8b 05 eb 2e 00 00  mov    0x2eeb(%rip),%rax        # 404030 <stdin@@GLIBC_2.2.5>
  401145:       |         48 89 c7              mov    %rax,%rdi
  401148:       |         e8 f3 fe ff ff        call   401040 <fgetc@plt>
  40114d:       |         89 45 fc              mov    %eax,-0x4(%rbp)
/tmp/test.c:10
  401150:       |         83 7d fc ff           cmpl   $0xffffffff,-0x4(%rbp)
  401154:       |  /----- 74 22                 je     401178 <main+0x42>
/tmp/test.c:13
  401156:       |  |      83 7d fc 6a           cmpl   $0x6a,-0x4(%rbp)
  40115a:       |  |  /-- 75 0a                 jne    401166 <main+0x30>
/tmp/test.c:14
  40115c:       |  |  |   bf 10 20 40 00        mov    $0x402010,%edi
  401161:       |  |  |   e8 ca fe ff ff        call   401030 <puts@plt>
/tmp/test.c:16
  401166:       |  |  \-> 83 7d fc 6b           cmpl   $0x6b,-0x4(%rbp)
  40116a:       +--|----- 75 d2                 jne    40113e <main+0x8>
/tmp/test.c:17
  40116c:       |  |      bf 15 20 40 00        mov    $0x402015,%edi
  401171:       |  |      e8 ba fe ff ff        call   401030 <puts@plt>
/tmp/test.c:7
  401176:       \--|----- eb c6                 jmp    40113e <main+0x8>
/tmp/test.c:11
  401178:          \----> 90                    nop
/tmp/test.c:20
  401179:                 b8 00 00 00 00        mov    $0x0,%eax
/tmp/test.c:21
  40117e:                 c9                    leave  
  40117f:                 c3                    ret 
```

And match doesn't work on following line:
```
  40116a:       +--|----- 75 d2                 jne    40113e <main+0x8>
```

So, I would suggest to add `+` symbol to the match. Also, it would be better to use non-capturing group here:
```suggestion
    asmOpcodeRe = /^\s*(?<address>[\da-f]+):(?:[|\\\/>X+-]|\s)*(?<opcodes>([\da-f]{2} ?)+)\s*(?<disasm>.*)/;
```